.github/workflow/tests.yml

```
@@ -0,0 +1,27 @@
+name: Test ocrd_cis installation and run tests
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
```

I would not be so restrictive here:
```suggestion
  push:
  pull_request:
  workflow_dispatch:
```

---

You were faster than me. I was already working on a fix since the workflow was not triggered for PRs.